restmanager/restmanager/settings.py

```diff
@@ -76,14 +76,26 @@
 
 # Database
 # https://docs.djangoproject.com/en/3.0/ref/settings/#databases
-
-DATABASES = {
+# SQLite
```

Comment:
I thought we were not using sqlite? why is it here? some use cases still?

Reply:
Local testing only